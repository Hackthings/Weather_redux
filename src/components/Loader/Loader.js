import React, { PureComponent } from 'react';
import SvgIcon from '../../components/SvgIcon';

import loader from '../../resources/Spinner-1s-200px.svg';


const Loader = () =>{
    return(
        <div className="Loader">
            <SvgIcon className="Loader__gif" src={loader} />
        </div>
    )
}

export default Loader