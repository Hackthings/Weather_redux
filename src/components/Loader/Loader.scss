.Loader{
    display: flex;
    justify-content: center;
    &__gif{
        width: 200px;        
    }
}