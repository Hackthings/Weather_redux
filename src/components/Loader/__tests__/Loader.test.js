import React from 'react';
import Loader from '../Loader';
import { shallow, mount } from 'enzyme';

describe('<Loader/>', () => {
    it('should render properly', () => {
        const loader = shallow(<Loader />);
        expect(loader).toMatchSnapshot();
    })
})