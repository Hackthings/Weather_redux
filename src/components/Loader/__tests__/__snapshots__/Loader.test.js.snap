// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Loader/> should render properly 1`] = `
<div
  className="Loader"
>
  <Icon
    className="Loader__gif"
    src="test-mock"
  />
</div>
`;
